services/notification/notifier.go

```diff
@@ -32,15 +32,19 @@ type (
 	}
 )
 
-var _ base.Notifier = &notificationService{}
+var _ notify.Notifier = &notificationService{}
 
 // NewNotifier create a new notificationService notifier
-func NewNotifier() base.Notifier {
+func NewNotifier() notify.Notifier {
 	ns := &notificationService{}
 	ns.issueQueue = queue.CreateQueue("notification-service", ns.handle, issueNotificationOpts{})
 	return ns
 }
 
+func init() {
+	notify.RegisterNotifier(NewNotifier())
+}
```

Comment:
I don't see an equivalent `import _ "…/services/notification"` registered in the tests, which is probably why the CI fails.

Reply:
The test failure is notification depends on `queue` which depend on `setting` so that it cannot be init in `init` function. Resolved.